types/networkparams.go

```diff
@@ -55,7 +55,9 @@ func (p *NetworkParams) GetSecondaryEthereumConfig() (*vega.SecondaryEthereumCon
 	param := netparams.BlockchainsSecondaryEthereumConfig
 	val, ok := p.Params[param]
 	if !ok {
-		return nil, fmt.Errorf("failed to get EthereumConfig, missing '%s' network parameter", param)
+		return &vega.SecondaryEthereumConfig{}, nil
```

Comment:
Just a heads up that the name of this has changed after a code review of the Vega branch:
https://github.com/vegaprotocol/vega/pull/10632#issuecomment-2020820183

its now `&vega.EVMChainConfig{}`